bokeh/colors/hsl.py

```diff
@@ -138,7 +138,7 @@ def to_rgb(self):
             :class:`~bokeh.colors.rgb.RGB`
 
         '''
-        from .rgb import RGB # prevent circular import
+        from .rgb import RGB  # prevent circular import
```

Comment:
This extra space is not a mistake?

Reply:
Not AFAIK, two spaces before a comment seems to be common in several tools at this point.